src/intl/en/page-layer-2.json

```
@@ -27,10 +27,10 @@
   "layer-2-what-is-layer-1-list-link-2": "Learn what Ethereum is.",
   "layer-2-why-do-we-need-layer-2-title": "Why do we need layer 2?",
   "layer-2-why-do-we-need-layer-2-1": "Three desirable properties of a blockchain are that it is <strong>decentralized, secure, and scalable</strong>. The <a href=\"https://www.ledger.com/academy/what-is-the-blockchain-trilemma\" target=\"_blank\">blockchain trilemma</a> states that a simple blockchain architecture can only achieve two out of three. Want a secure and decentralized blockchain? You need to sacrifice scalability.",
-  "layer-2-why-do-we-need-layer-2-2": "Ethereum has reached the network's current capacity with <a href=\"https://etherscan.io/chart/tx\" target=\"_blank\">1+ million transactions per day</a> and high demand for each of these transactions. The success of Ethereum and the demand to use it has caused gas prices to rise substantially. Therefore the need for scaling solutions has increased in demand as well. This is where layer 2 networks come in.",
+  "layer-2-why-do-we-need-layer-2-2": "Ethereum currently processes <a href=\"https://etherscan.io/chart/tx\" target=\"_blank\">1+ million transactions per day</a>. The demand for transactions to be included in blocks can cause gas prices be high. This is where layer 2 networks come in.",
```

```suggestion
  "layer-2-why-do-we-need-layer-2-2": "Ethereum currently processes <a href=\"https://etherscan.io/chart/tx\" target=\"_blank\">1+ million transactions per day</a>. The demand to use Ethereum can cause transaction fee prices be high. This is where layer 2 networks come in.",
```
@jmcook1186 i think we need some framework how to tackle jargon.. when to be specific and use terms like gas or blocks and when to use normal language like transaction fee

on pages like L2 i suspect we should attempt to remove this jargon as its audience probably doesnt need to know the precise details or terms.  This is a discussion tho and curious about other's take. 

cc @minimalsm